.github/workflows/dependency-review.yml

```diff
@@ -30,6 +30,7 @@ jobs:
         uses: actions/checkout@v4
       - name: "Dependency Review"
         uses: actions/dependency-review-action@v4
+        continue-on-error: true
```

Comment:
Temporarily made dependency review optional. We hit github's summary size limit, which (according to ai) can't be increased. I'll revert this change after merging this PR

<img width="1198" alt="image" src="https://github.com/user-attachments/assets/61eb5f03-ca1b-4781-8236-e6c3b750329a" />